apps/web/providers/PostHogProvider.tsx

```diff
@@ -0,0 +1,58 @@
+// app/providers.tsx
+"use client";
+import { useUser } from "@clerk/nextjs";
+import { usePathname, useSearchParams } from "next/navigation";
+import posthog from "posthog-js";
+import { PostHogProvider } from "posthog-js/react";
+import { useEffect } from "react";
+if (typeof window !== "undefined") {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+    api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST!,
+    capture_pageview: false,
+    disable_session_recording: false,
+  });
+}
+
+export function PostHogPageview(): JSX.Element {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
+  useEffect(() => {
+    if (pathname) {
+      let url = window.origin + pathname;
+      if (searchParams?.toString()) {
+        url = url + `?${searchParams.toString()}`;
+      }
+      posthog.capture("$pageview", {
+        $current_url: url,
+      });
+    }
+  }, [pathname, searchParams]);
+
+  return <></>;
+}
+
+export function PostHogAuthIdentify(): JSX.Element {
+  const { user, isLoaded } = useUser();
+
+  useEffect(() => {
+    if (!isLoaded) {
+      return;
+    }
+    if (user) {
+      posthog.get_distinct_id();
+      posthog.alias(user.id, posthog.get_distinct_id());
```

Comment:
I have no idea about this, but it looks like a mistake, why do we get 2 distinct ids?